### PR TITLE
Adds dark background for media richlinks in opinion articles

### PR DESCRIPTION
**Pull request by Joshua-David**
## What does this change?
Adds dark background for media richlinks in opinion articles so they are legible 

Before:
screen shot 2018-02-02 at 11 43 43

After:
screen shot 2018-02-02 at 11 44 02

### CODE REVIEW

**Comment by NataliaLKB** [comment]
@Joshua-David you need to rebase or merge with master for the tests to pass in teamcity

**Comment by Joshua-David** [comment]
👍 will do on all my PR's

**Comment by Joshua-David** [comment]
@NataliaLKB have pulled master in, checks seem fine now

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 1]
```diff
@@ -850,6 +850,10 @@ $quote-mark: 35px;
             background-color: $garnett-neutral-5;
         }
 
+        .rich-link.tone-media--item {
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content-garnett/_types.scss]: This should probably be in this file: https://github.com/guardian/frontend/blob/dae274df3741ce52c1bf5b38e3482d294b0685e8/static/src/stylesheets/module/onward-garnett/_rich-links-enhanced.scss

**Comment by Joshua-David** [comment]
@NataliaLKB good call

**Review by NataliaLKB — APPROVED** [review]

File: static/src/stylesheets/module/onward-garnett/_rich-links-enhanced.scss [hunk 2]
```diff
@@ -106,6 +106,20 @@
...
+// Opinion article specific styles
+//-----------------------
+
+.content--type-guardianview,
+.content--type-comment {
+    .rich-link {
+        background-color: $garnett-neutral-5;
+    }
+
+    .rich-link.tone-media--item {
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/onward-garnett/_rich-links-enhanced.scss]: Why are we still using tones for media items? I thought one of the aims of the redesign is that we could start to get rid of tones?

**Pull request merged**